docs/user/abandoned-projects.rst

```diff
@@ -3,6 +3,8 @@ Policy for Abandoned Projects
 
 This policy describes the process by which a Read the Docs project name may be changed.
 
+.. tip:: If you are reporting a project but a name change does not solve the issue, please see :doc:`/unofficial-projects`.
```

Comment:
```suggestion
.. tip:: If you want to de-list a project's fork from search results, please see :doc:`/unofficial-projects`.
```

Reply:
Somewhat the same but without "fork" and "de-list"?

```suggestion
.. tip:: If you are requesting to remove or suppress an unofficial project, please see :doc:`/unofficial-projects`.
```

Reply:
I don't think your suggestion is correct. This policy wont "remove or suppress", it will de-list it from search results.

Reply:
I get that - the tip doesn't say what the policy does, it is a hint for people that would be interested in "requesting to remove or suppress an unofficial project" -- if they navigate to the article, they can see exactly what de-list means and what the policy is about.

Reply:
Why don't we tell the user what's the policy about in the tip instead so they know what's it about without having to click on it and read the policy?

Reply:
Well in a tip box or any other navigation or admonition, it's a matter of _discoverability_ - to me, this means to use words that appeal to the user in order to get them to the right final destination. That's why I would slightly rephrase what the policy exactly is about in order to catch the widest-possible variety of user journeys.

For instance, a user may think that the project is _abandoned_ and should be _removed_. So now the user is reading the "Policy for Abandoned Projects", but then they see the tip box and go "aha, maybe it's this one".

In your first suggestion, it would be easy to answer "no" to the question ("If you want to de-list a project's fork from search results, please see ...")

So instead of suggestions to narrow down or be very specific about the policy, I would suggest that we have a one-liner that can really catch a wide variety of user intentions _prior_ to reading the Policy. Of course not in a misleading or inaccurate way!

I hope that makes sense?

Reply:
Manuel is correct. Your suggested wording is needlessly indirect, and we should be explicit where possible.

Reply:
I don't find any reason to discuss it further. My previous comment elaborates why I would use different terms in the tip box.

I'll merge @humitos' suggestion then.

Reply:
We don't have any policy that removes a project from our site, so using that language is definitely not a reasonable expectation to set. Bringing a wider set of people to a page that won't be well served by the policy seems like an anti-pattern, so I'm not quite sure the argument for it.

Reply:
I'll try to add a comment to future tip-boxes so the purpose of the tip box is understood.